### PR TITLE
Complete CSS Layout Classes exercise.

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
Screenshot 2023-07-20 at 4 15 55 PM

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>
```
Comment (shawnkost) [inline css-layout-classes/index.html]: If you're going to use a `header` here, you should remove the `div` and give the `column-full` class to the `header`

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" alt="yoshi" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-fourth">
+          <img src="images/kirby.png" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: Needs an `alt` attribute

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/fox.png" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: Needs an `alt` attribute

File: css-layout-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: Needs an `alt` attribute

File: css-layout-classes/index.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/ness.png" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: Needs an `alt` attribute

File: css-layout-classes/index.html [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+            additional spin-off titles of various genres.
+          </p>
+          <p>
+            The games of the first genre are mostly single-screen
+            platform/action puzzle types, featuring Donkey Kong as the opponent
+            in an industrial construction setting.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/dk.png" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: Needs an `alt` attribute

File: css-layout-classes/index.html [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/mario.png" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: Needs an `alt` attribute

File: css-layout-classes/styles.css [hunk 8]
```diff
@@ -0,0 +1,27 @@
+.container {
+  max-width: 800px;
+  margin: 0, auto;
```
Comment (shawnkost) [inline css-layout-classes/styles.css]: This syntax is incorrect. There should be no commas. It should just be `margin: 0 auto`

File: css-layout-classes/styles.css [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-basis: 100%;
```
Comment (shawnkost) [inline css-layout-classes/styles.css]: `flex-basis` is a child property, so it doesn't get applied to flex parents. This has no impact, so it should be removed

File: css-layout-classes/styles.css [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+.column-one-third {
+  width: calc(100% / 3);
+  flex-basis: auto;
+  justify-content: space-evenly;
+}
+
+.column-two-thirds {
+  width: calc(100% / 2);
+  flex-basis: auto;
+}
```
Comment (shawnkost) [inline css-layout-classes/styles.css:30]: Missing a selector for `column-one-fourth`

Comment (shawnkost) [inline css-layout-classes/styles.css:30]: Missing a selector for `column-full`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Dennis!

**Comment by dennis-pnguyen** [comment]
Added alt attributes and removed initial div

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**